<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasureDialog</class>
 <widget class="QWidget" name="MeasureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeasureDialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>161</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>测量列表</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>第一个点</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_point1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>202</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>第二个点</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_point2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>202</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>距离</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_value">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>270</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
